src/manager.cpp

```diff
@@ -120,6 +121,31 @@ sdbusplus::async::task<> Manager::startSyncEvents()
     co_return;
 }
 
+void Manager::syncData(const config::DataSyncConfig& dataSyncCfg)
+{
+    using namespace std::string_literals;
+    std::string syncCmd{"rsync --archive --compress"};
+
+    // Add source data path
+    syncCmd.append(" "s + dataSyncCfg._path);
+
+#ifdef UNIT_TEST
+    syncCmd.append(" "s);
+#else
+    // TODO Support for remote (i,e sibling BMC) copying needs to be added.
+#endif
+
+    // Add destination data path
+    syncCmd.append(dataSyncCfg._destPath.value_or(dataSyncCfg._path));
+    int result = std::system(syncCmd.c_str()); // NOLINT
+    if (result != 0)
```

Comment:
@RameshIyyar Do you have any plans for any sort of a flight recorder to keep a port of the sync history?

Reply:
Yes Matt that's the last piece I plan to include.